### ISSUE TITLE
Achievement glitch

### ISSUE THREAD

**Issue by Zelgaillen** [dalaranwow/dalaran-wow#5693]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)

**Expected behaviour**: Tell us what should happen instead

**Steps to reproduce the problem**:

1. 
2. 
3. 

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by Zelgaillen** [comment]
I just completed Naxx 10-man for the first time on this character. Months ago I had completed Naxx 25-man and Ulduar 25-man received the achievements and moved on to other raids/unlocked the last 5-man dungeons etc.

I should still have the 25-man achievements for naxx & Uld however now I have the completion for 10-man Uld & Naxx although I have never completed 10-man Uld. Basically, completing 10man naxx overwrote my prior achievements as 10-man.

I cannot get a GM to answer me this is driving me crazy.

**Comment by Golganeth** [comment]
Do you have any kind of proof? Please open an in-game ticket and we will help you out.
Closing

**Comment by Zelgaillen** [comment]
Hey, I do not have any proof such as screen shots etc. I was hoping that
there would be a back log for the achievement, some sort of record. I did
setup an in game ticket, however I did not receive a response, but I can
try again.

Thank you for your reply. Do you know of any potential way to research
proof of this error occurring, also have you ever heard of this happening
before? I'm really bummed this happened but I know it's hard to catch every
bug.

I wonder if the issue is that I did the 25mans before I did a 10 man which
is pretty backwards.

Thanks,
Tyler

**Comment by Golganeth** [comment]
I've personally never seen this issue happen before, which leads me to think it must be something very isolated that only happened to you, that's why I asked for some kind of proof to figure out some things.

Don't worry, if what you say is truth, it's very easy for us to verify it (character statistics in-game, logs, etc.) so keep that in-game ticket up and send us an e-mail to "dalaran.community@gmail.com" explaining as much as you can about your problem (if it was all on the same day, if achievements disappeared instantly or not, if there was a server crash, if you got any gear from the raids or quests done...) and we will check it out and try to help you as soon as possible.

Best regards!

**Comment by Zelgaillen** [comment]
Thank you very much! I will email to the community gmail and setup another
in-game ticket. I really appreciate it!

**Comment by Zelgaillen** [comment]
Hey, so I sent the ticket in-game, and emailed to
dalaran.community@gmail.com. I received a notice in my in-game mailbox 8
days ago that they would be forwarding my ticket along, My ticket was
closed, and I haven't heard back yet. I don't want to be annoying, but I
was really hoping we could get this issue resolved, and I'm worried that
the ticket was lost somehow, Can you check to see if anyone is working on
it?

Thank you for your time, Please respond to let me know,
-Tyler
